BookingsApi/BookingsApi.DAL/Queries/GetAllocationHearingsBySearchQuery.cs

```diff
@@ -65,6 +70,13 @@ public async Task<List<VideoHearing>> Handle(GetAllocationHearingsBySearchQuery
             
             if (!_isTest)
                 hearings = hearings.Where(h1 => h1.CaseTypeId != 3); //exclude generic type test cases from prod
+
+            if (query.IncludeWorkHours)
+            {
+                hearings = hearings.Include(h => h.Allocations).ThenInclude(a => a.JusticeUser)
+                    .ThenInclude(x => x.VhoWorkHours)
+                    .Include(h => h.Allocations).ThenInclude(a => a.JusticeUser).ThenInclude(x => x.VhoNonAvailability);
```

Comment:
have you checked SQL generated by this query? It might be worth splitting this out

Reply:
Yeah it actually came up as a warning. I did try to split it but the warning didn't disappear. I'll push a change and we can go over it